config/bidderinfo.go

```diff
@@ -356,6 +358,50 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
```

Comment:
We are validating aliasBidder capabilities here. I think it is more intuitive to use `aliasBidder` or `aliasBidderInfo` in function arguments. 
```
func validateAliasCapabilities(aliasBidderInfo BidderInfo, infos BidderInfos, bidderName string) error {
```